chart/templates/postgres-exporter-promrule.yaml

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE.
+# This file is generated from scripts/sync-postgres-exporter-promrule.sh
```

Comment:
Is this correct? Isn't `alerts/postgres-exporter-alerts.yaml` the generated one?

Reply:
Yeah it's left over from some older code, I will remove it.